### PR TITLE
chore(engine): add migration note for changed job prio defaults

### PR DESCRIPTION
**Pull request by mboskamp**
related to camunda/camunda-bpm-platform/issues/3066

### CODE REVIEW

File: content/reference/deployment-descriptors/tags/process-engine.md [hunk 1]
```diff
@@ -626,7 +639,7 @@ The following is a list with the most commonly used process engine configuration
         You can combine this property with <code>jobExecutorPriorityRangeMax</code> to specify a job priority range for the job executor.
         Note, that you can configure the priority of batch jobs and history cleanup jobs via the <code>batchJobPriority</code> and <code>historyCleanupJobPriority</code> properties. Both default to <code>0</code>.
       <p>
-        <strong>Default value:</strong> <code>0</code>
+        <strong>Default value:</strong> <code><code>-2<sup>63</sup>-1</code></code> (`Long.MIN_VALUE`)
```
Comment (danielkelemen) [inline content/reference/deployment-descriptors/tags/process-engine.md:642]: The backticks don't seem to work in the table and looks like [long min](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html) is _just_ -2<sup>63</sup>

```suggestion
        <strong>Default value:</strong> <code><code>-2<sup>63</sup></code></code> (<code>Long.MIN_VALUE</code>)
```

File: content/reference/deployment-descriptors/tags/process-engine.md [hunk 2]
```diff
@@ -638,7 +651,7 @@ The following is a list with the most commonly used process engine configuration
         You can combine this property with <code>jobExecutorPriorityRangeMin</code> to specify a job priority range for the job executor.
         Note, that you can configure the priority of batch jobs and history cleanup jobs via the <code>batchJobPriority</code> and <code>historyCleanupJobPriority</code> properties. Both default to <code>0</code>.
       <p>
-        <strong>Default value:</strong> <code>2<sup>63</sup>-1</code>
+        <strong>Default value:</strong> <code>2<sup>63</sup>-1</code> (`Long.MAX_VALUE`)
```
Comment (danielkelemen) [inline content/reference/deployment-descriptors/tags/process-engine.md:654]:
```suggestion
        <strong>Default value:</strong> <code>2<sup>63</sup>-1</code> (<code>Long.MAX_VALUE</code>)
```

File: content/update/patch-level.md [hunk 3]
```diff
@@ -363,7 +363,14 @@ These patches replace the AngularJS libraries with XLTS for AngularJS. Where Ang
...
+Note: this behavior is slightly changed with the [next patch]({{< ref "#job-executor-priority-range-properties-default-changed" >}}).
+
+## 7.18.2 to 7.18.3 / 7.17.8 to 7.17.9
+
+### Job executor priority range properties default changed
+
+The default value for the job executor priority range property [jobExecutorPriorityRangeMin]({{< ref "/reference/deployment-descriptors/tags/process-engine.md#jobExecutorPriorityRangeMin" >}}) has changed from 0 to <code>-2<sup>63</sup>-1</code> (`Long.MIN_VALUE`). This property and its counterpart ([jobExecutorPriorityRangeMax]({{< ref "/reference/deployment-descriptors/tags/process-engine.md#jobExecutorPriorityRangeMax" >}})) only have an effect if [jobExecutorAcquireByPriority]({{< ref "/reference/deployment-descriptors/tags/process-engine.md#jobExecutorAcquireByPriority" >}}) is set to true.
```
Comment (danielkelemen) [inline content/update/patch-level.md:373]:
```suggestion
The default value for the job executor priority range property [jobExecutorPriorityRangeMin]({{< ref "/reference/deployment-descriptors/tags/process-engine.md#jobExecutorPriorityRangeMin" >}}) has changed from 0 to <code>-2<sup>63</sup></code> (`Long.MIN_VALUE`). This property and its counterpart ([jobExecutorPriorityRangeMax]({{< ref "/reference/deployment-descriptors/tags/process-engine.md#jobExecutorPriorityRangeMax" >}})) only have an effect if [jobExecutorAcquireByPriority]({{< ref "/reference/deployment-descriptors/tags/process-engine.md#jobExecutorAcquireByPriority" >}}) is set to true.
```

**Review by danielkelemen — APPROVED** [review]
Looks good, added 2 hints about long min and formatting.

**Comment by mboskamp** [comment]
Thanks for flagging this @danielkelemen, I have adjusted the other two PRs as well.

**Pull request merged**